guides/common/modules/proc_adding-a-bonded-interface.adoc

```diff
@@ -39,33 +39,34 @@ endif::[]
 +
 [options="nowrap", subs="verbatim,quotes,attributes"]
 ----
-# hammer host create --name bonded_interface \
---hostgroup-id _My_Host_Group_ID_ \
---ip=_192.168.100.123_ \
---mac=_52:54:00:14:92:2a_ \
---subnet-id=_My_Subnet_ID_ \
+# hammer host create \
+--ask-root-password yes \
+--hostgroup _My_Host_Group_ \
+--ip=_My_IP_Address_ \
+--mac=_My_MAC_Address_ \
 --managed true \
-   --interface="identifier=_eth1_, \
-               mac=_52:54:00:62:43:06_, \
+   --interface="identifier=_My_NIC_1, \
+               mac=_My_MAC_Address_1_, \
                managed=true, \
                type=Nic::Managed, \
                domain_id=_My_Domain_ID_, \
                subnet_id=_My_Subnet_ID_" \
-   --interface="identifier=_eth2_, \
-               mac=_52:54:00:d3:87:8f_, \
+   --interface="identifier=_My_NIC_2_, \
+               mac=_My_MAC_Address_2_, \
                managed=true, \
                type=Nic::Managed, \
                domain_id=_My_Domain_ID_, \
                subnet_id=_My_Subnet_ID_" \
    --interface="identifier=_bond0_, \
-               ip=_172.25.18.123_, \
+               ip=_My_IP_Address_2_, \
                type=Nic::Bond, \
                mode=active-backup, \
-               attached_devices=[_eth1,eth2_], \
+               attached_devices=[_My_NIC_1_,_My_NIC_2_], \
                managed=true, \
                domain_id=_My_Domain_ID_, \
                subnet_id=_My_Subnet_ID_" \
```

Comment:
I'd say this probably doesn't need indenting, at least not the first level. What do you think?